lib/Convos.pm

```diff
@@ -150,6 +142,18 @@ sub _home_relative_to_lib {
   $self->static->paths->[0] = $self->home->rel_dir('public');
 }
 
+sub _plugins {
+  my $self    = shift;
+  my $plugins = $self->config('plugins');
+
+  unshift @{$self->plugins->namespaces}, 'Convos::Plugin';
+
+  $ENV{CONVOS_PLUGINS} //= $plugins ? '' : 'Sharelog';
+  $plugins ||= {};
+  $plugins->{$_} = {} for split /,/, $ENV{CONVOS_PLUGINS};
+  $self->plugin($_ => $plugins->{$_}) for grep { $plugins->{$_} } keys %$plugins;
```

Comment:
Yeah, it has to be this way. After all, you still have `$app->core` available, so you can do whatever.